### PR TITLE
updated to 3.0.0

### PR DESCRIPTION
**Pull request by sudoku-lord**
Fixes some typos in the README. I ran every example and they all work fine, so... I think we're alright?

### CODE REVIEW

File: examples/postcards_template.rb [hunk 1]
```diff
@@ -61,32 +61,38 @@
...
+    "address_line1": "120 6th Ave",
+    "address_city": "Boston",
+    "address_state": "MA",
+    "address_country": "US",
+    "address_zip": 12345
+  }
 )
 
 # send a postcard
 pp lob.postcards.create(
```
Comment (ajorczak-lob) [inline examples/postcards_template.rb:87]: You'll want to remove this debugging code.

File: lib/lob/resources/resource_base.rb [hunk 2]
```diff
@@ -1,6 +1,7 @@
 require "rest-client"
 require "json"
 require "uri"
+require "pp"
```
Comment (ajorczak-lob) [inline lib/lob/resources/resource_base.rb:4]: Let's remove this debugging require

File: lib/lob/resources/resource_base.rb [hunk 3]
```diff
@@ -82,6 +83,8 @@ def submit(request)
             response = RestClient.send(method, url, body, headers)
           end
 
+          pp JSON.parse(response)
```
Comment (ajorczak-lob) [inline lib/lob/resources/resource_base.rb:86]: Debugging code.

File: lib/lob/resources/resource_base.rb [hunk 4]
```diff
@@ -12,7 +13,7 @@ def initialize(config)
         @config = config
       end
 
-      def list(query={})
+      def list(**query)
```
Comment (ajorczak-lob) [inline lib/lob/resources/resource_base.rb:16]: This is actually doing two different things here:

`**query` is similar to node's `...query` in expressing variable arity. `query = {}` is assigning a default value to query if it's undefined. Not really sure what's changed here but I'll take a look locally to verify.

File: lib/lob/resources/resource_base.rb [hunk 5]
```diff
@@ -139,4 +142,4 @@ def resource_url(resource_id)
 
     end
   end
-end
+end
```
Comment (ajorczak-lob) [inline lib/lob/resources/resource_base.rb:145]: Let's add a new line here

File: .travis.yml [hunk 6]
```diff
@@ -3,11 +3,12 @@ language: ruby
...
   - jruby-9.1.9.0
   - jruby-9.2.9.0
-  - 2.4
   - 2.5
   - 2.6
   - 2.7
+  - 3.0
 env:
   JRUBY_OPTS=--2.0
+  RUBYOPT="-W0"
```
Comment (ajorczak-lob) [inline .travis.yml:12]: This is setting verbose mode to "silent". Is that what we want to do here?

File: .travis.yml [hunk 7]
```diff
@@ -3,11 +3,12 @@ language: ruby
 rvm:
   - jruby-9.1.9.0
   - jruby-9.2.9.0
-  - 2.4
   - 2.5
```
Comment (ajorczak-lob) [inline .travis.yml:6]: Let's take out 2.5 as well. EOL was 3-31-2021

**Review by ajorczak-lob — CHANGES_REQUESTED** [review]
Some debugging code should be removed before we publish. Also, we'll want to clean up some of the testing options (RUBYOPT question and we can also remove Ruby 2.5)

File: examples/postcards_template.rb (hunk 1 above)
Comment (sudoku-lord) [reply examples/postcards_template.rb:87]: If you're referring to the usage of `pp`, that one is actually a replacement for `puts`, which was yielding pretty unreadable output. It's not for debugging purposes.

File: lib/lob/resources/resource_base.rb (hunk 4 above)
Comment (sudoku-lord) [reply lib/lob/resources/resource_base.rb:16]: This is actually a leftover from something I was experimenting with. I'll change it back.

File: .travis.yml (hunk 6 above)
Comment (sudoku-lord) [reply .travis.yml:12]: I was trying to silence the many, many warnings I've been getting about URI, but this didn't work. Do you have advice about a better solution?

**Review by ajorczak-lob — APPROVED** [review]
Looks good. Thanks for the updates!

**Pull request merged**